packages/neuron-ui/config-overrides.js

```diff
@@ -1,4 +1,7 @@
-module.exports = function override(config, env) {
-  //do stuff with the webpack config...
-  return config;
-}
\ No newline at end of file
+const path = require('path')
+
+module.exports = function override(config) {
+  const webpackCofnig = { ...config }
```

Comment:
It seems meanless here.

Reply:
Can't get through ESLint without doing this.

Reply:
I see, don't change the argument.